downloader/src/main/kotlin/Main.kt

```diff
@@ -35,13 +35,13 @@ import com.here.ort.model.MergedAnalyzerResult
 import com.here.ort.model.Package
 import com.here.ort.model.RemoteArtifact
 import com.here.ort.model.VcsInfo
+import com.here.ort.utils.encodeOrUnknown
```

Comment:
Nit: This import should be further down, just where the import of fileSystemEncode was before.